packages/contracts/contracts/core/plugin/PluginClones.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import "@openzeppelin/contracts/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts/utils/introspection/ERC165.sol";
+import "@openzeppelin/contracts/utils/Context.sol";
+
+import {PermissionManager} from "../permission/PermissionManager.sol";
+
+import {IDAO} from "../IDAO.sol";
+import {DaoAuthorizable} from "../component/DaoAuthorizable.sol";
+import {AppStorage} from "../../utils/AppStorage.sol";
+
+/// @title PluginClones
+/// @notice An abstract contract to inherit from when creating a proxy contract.
+/// This should be used to deploy EIP-1167 clones.
+abstract contract PluginClones is Initializable, ERC165, Context, AppStorage {
```

Comment:
If the other plugins are named "upgradable" I suggest we follow a similar naming (adjective) here. How about "clonable" or even "cloned" ?

Reply:
amended